Bluepill-cli/Bluepill-cli/Bluepill/BPExecutionContext.h

```diff
@@ -29,4 +30,6 @@
 // final exit status for this context run
 @property (nonatomic, assign) BPExitStatus finalExitStatus;
 
+- (BPWriter *)simulatorLogWriter;
```

Comment:
move to a property (just to be more consistent), ideally, there should be a builder for this and make everything immutable, but we can do that later.